doc/WebSite/Zero/Permission-Management.md

```diff
@@ -82,15 +82,15 @@ permission for a user:
         }
     }
 
-UserManager has many methods to control permissions for users. In this
-example, we're getting a UserId and PermissionName and using
+The UserManager has many methods to control the permissions of users. In this
+example, we're getting a UserId and PermissionName and using the
 UserManager's **ProhibitPermissionAsync** method to prohibit a
 permission for a user.
 
 When we **prohibit** a permission for a user, he/she **can not** be
 authorized for this permission even his/her roles are **granted** for
-the permission. We can say same principle for granting. When we
+the permission. We can use the same principle for granting permissions. When we
 **grant** a permission specifically for a user, this user **is granted**
-for the permission even roles of the user are not granted for the
-permission. We can use **ResetAllPermissionsAsync** for a user to delete
-all user-specific permission settings for the user.
+the permission even if the roles of the user are not granted for the
+permission. We can use the **ResetAllPermissionsAsync** method for a user to delete
+all user-specific permission settings for them.
```

Comment:
"We can use the ResetAllPermissionsAsync method ~`for a user`~ to delete all user-specific permission settings for `a user`".